pkg/descheduler/utils/priority.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2022 The Koordinator Authors.
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"fmt"
+
+	schedulingv1 "k8s.io/client-go/listers/scheduling/v1"
+
+	deschedulerconfig "github.com/koordinator-sh/koordinator/pkg/descheduler/apis/config"
+)
+
+const SystemCriticalPriority = 2 * int32(1000000000)
```

Comment:
What is the difference between system-cluster-critical and this one ?

Reply:
It's same value. Updated